packages/react-native-reanimated/Common/cpp/worklets/SharedItems/Shareables.cpp

```diff
@@ -233,7 +233,7 @@ ShareableObject::ShareableObject(
 jsi::Value ShareableObject::toJSValue(jsi::Runtime &rt) {
   auto obj = jsi::Object(rt);
   for (size_t i = 0, size = data_.size(); i < size; i++) {
-    obj.setProperty(rt, data_[i].first.c_str(), data_[i].second->toJSValue(rt));
+      obj.setProperty(rt, jsi::String::createFromUtf8(rt, data_[i].first), data_[i].second->toJSValue(rt));
```

Comment:
Let's restore the original indentation.
```suggestion
    obj.setProperty(rt, jsi::String::createFromUtf8(rt, data_[i].first), data_[i].second->toJSValue(rt));
```

Reply:
Oh right! Too much prettier for me 😅